russian/scratch/open-close/open-close.md

```diff
@@ -1 +1,5 @@
 Вы поморгали несколько раз, зефир поменял цвет на зеленый. При этом вы не можете вспомнить, каким он был до того.
+
+Внезапно Вы услышали нарастающий гул снаружи. Все началось с легкой вибрации но очень быстро стены стали ходить ходуном.
+Еще мновение и Вы будете погребены под грудами обломков стен из зефира и несущих консрукций из печеньки. 
+Вы еще можете [предпринять что-нибудь](../../do-something/do-something.md) пока не упал потолок, иначе все что останеться - [прогрызать путь к свободе](../../eating-walls/eating-marshmallows.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.